src/components/equitySection/equitySection.js

```diff
@@ -6,30 +6,30 @@ import { getImage } from "gatsby-plugin-image";
 import { BgImage } from "gbimage-bridge";
 import useQuoteImages from "../../hooks/useQuoteImages";
 
-import content from "../../pages/equity.content.yml";
+import indexContent from "../../pages/index.content.yml";
+import stateContent from "../../pages/state/[state].content.yml";
 
 import * as styles from "./equitySection.module.css";
 
-export default function EquitySection({ size = "lg" }) {
+export default function EquitySection({ page = "main"}) {
   const [{ quoteImagesByPath }] = useQuoteImages();
 
   const {
-    topQuote: { image },
-    embedded: { description, heading, link },
-  } = content;
+    equityEmbed: { image, description, heading, link },
+  } = page === "main" ? indexContent : stateContent;
 
   return (
-    <Row className={styles[size]}>
+    <Row className={styles[page === "main" ? "lg" : "sm"]}>
       <BgImage
         className={styles.background}
-        image={getImage(quoteImagesByPath[image])}
+        image={getImage(quoteImagesByPath[page !== "main" ? image.slice(3, image.length) : image])}
```

Comment:
What happening with the `slice`?

Reply:
The path from the state.js file is ../../images but the path as defined in the hook is just ../images. A better solution would be pathing from src everywhere. But at this point, maybe we should create that as a separate ticket if at all?

Reply:
💯  makes sense and agree